tensorflow/lite/micro/examples/hello_world/evaluate.py

```diff
@@ -0,0 +1,142 @@
+def generate_random_input(sample_count=1000):
+  # Generate a uniformly distributed set of random numbers in the range from
+  # 0 to 2π, which covers a complete sine wave oscillation
+  x_values = np.random.uniform(low=0, high=2 * np.pi,
+                               size=sample_count).astype(np.float32)
+  # Shuffle the values to guarantee they're not in order
+  np.random.shuffle(x_values)
+  return x_values
+
+
+# Get the metadata like scales and zero_points from the interpreter input/output
+# details.
+def get_quantization_params(interpreter_io_details):
+  quantize_params = interpreter_io_details.get('quantization_parameters')
+  scale = quantize_params.get('scales')
+  zero_point = quantize_params.get('zero_points')
+  return scale, zero_point
+
+
+# Invoke the tflm interpreter with x_values in the range of [0, 2*PI] and
+# returns the prediction of the interpreter.
+def get_tflm_prediction(hello_world_model_path, x_values):
+  # Create the tflm interpreter
+  tflm_interpreter = tflm_runtime.Interpreter.from_file(hello_world_model_path)
+
+  input_shape = np.array(tflm_interpreter.get_input_details(0).get('shape'))
+
+  y_predictions = np.empty(x_values.size, dtype=np.float32)
+  i = 0
+  for x_value in x_values:
+    y_predictions[i] = invoke_tflm_interpreter(input_shape,
+                                               tflm_interpreter,
+                                               x_value,
+                                               input_index=0,
+                                               output_index=0)
+    i += 1
+  return y_predictions
+
+
+# Invoke the tflite interpreter with x_values in the range of [0, 2*PI] and
+# returns the prediction of the interpreter.
+def get_tflite_prediction(hello_world_model_path, x_values):
+  # TFLite interpreter
+  tflite_interpreter = tf.lite.Interpreter(
+        model_path=hello_world_model_path,
+        experimental_op_resolver_type= \
+          tf.lite.experimental.OpResolverType.BUILTIN_REF)
+  tflite_interpreter.allocate_tensors()
+
+  input_details = tflite_interpreter.get_input_details()[0]
+  output_details = tflite_interpreter.get_output_details()[0]
+  input_shape = np.array(input_details.get('shape'))
+
+  y_predictions = np.empty(x_values.size, dtype=np.float32)
+  i = 0
+  for x_value in x_values:
+    y_predictions[i] = invoke_tflite_interpreter(input_shape,
+                                                 tflite_interpreter, x_value,
+                                                 input_details['index'],
+                                                 output_details['index'])
+    i += 1
+  return y_predictions
+
+
+def main(_):
+  hello_world_no_quant_model_path = os.path.join(PREFIX_PATH,
+                                                 'hello_world.tflite')
+
+  x_values = generate_random_input()
+
+  # Calculate the corresponding sine values
+  y_true_values = np.sin(x_values).astype(np.float32)
+
+  if _USE_TFLITE_INTERPRETER.value:
```

Comment:
Do we want this to be an addition instead of substitution? i.e. if `use_tflite` is set, run both TFL and TFLM, instead of just TFL.